samples/msal-core-samples/VanillaJSTestApp/app/ie11-sample/authConfig.js

```diff
@@ -0,0 +1,31 @@
+// Config object to be passed to Msal on creation
+var msalConfig = {
+    auth: {
+        clientId: "4b0db8c2-9f26-4417-8bde-3f0e3656f8e0",
+        redirectUri: "http://localhost:30662/"
+    },
+    cache: {
+        cacheLocation: "localStorage", // This configures where your cache will be stored
+        storeAuthStateInCookie: true, // Set this to "true" if you are having issues on IE11 or Edge
+    },
+    system: {
+        telemetry: {
```

Comment:
Telemetry doesn't need to be included, as it is an advanced feature.

Reply:
Sounds good, removed!